packages/caliper-cli/lib/bindCommand.js

```diff
@@ -35,11 +31,11 @@ module.exports.builder = function (yargs){
     yargs.options({
         'caliper-bind-sut' : {describe: 'The name of the platform to bind to', type: 'string' },
         'caliper-bind-sdk'  : {describe: 'Version of the platform SDK to bind to', type: 'string'},
-        'caliper-bind-cwd'  : {describe: 'The working directory for performing the SDK install', type: 'string'},
-        'caliper-bind-args'  : {describe: 'Additional arguments to pass to "npm install". Use the "=" notation when setting this parameter', type: 'string'}
+        'caliper-bind-cwd'  : {describe: 'The working directory for performing the SDK install', type: 'string' },
+        'caliper-bind-args'  : {describe: 'Additional arguments to pass to "npm install". Use the "=" notation when setting this parameter', type: 'string' },
+        'caliper-bind-file'  : {describe: 'Yaml file to override default (supported) package versions when binding an SDK', type: 'string' }
     });
     yargs.usage('Usage:\n  caliper bind --caliper-bind-sut fabric --caliper-bind-sdk 1.4.1 --caliper-bind-cwd ./ --caliper-bind-args="-g"');
-
     // enforce the option after these options
     yargs.requiresArg(['caliper-bind-sut','caliper-bind-sdk','caliper-bind-args', 'caliper-bind-cwd']);
```

Comment:
Hmm, the `requiresArg` call has been removed from our `run` command, so the options can also be specified from other sources (env, config file, etc). Strange how this line didn't cause a problem so far....

Reply:
the above are the required items as per the bind configuration only - which iirc also complains if the items are not passed. But you are correct - this is essentially a run command, so we should map to that pattern and only throw/reject if there are no values at the point of use